website/content/en/v0.32/upgrading/v1beta1-controller-policy.json

```diff
@@ -49,7 +49,8 @@
         "arn:${AWS_PARTITION}:ec2:${AWS_REGION}:*:instance/*",
         "arn:${AWS_PARTITION}:ec2:${AWS_REGION}:*:volume/*",
         "arn:${AWS_PARTITION}:ec2:${AWS_REGION}:*:network-interface/*",
-        "arn:${AWS_PARTITION}:ec2:${AWS_REGION}:*:launch-template/*"
+        "arn:${AWS_PARTITION}:ec2:${AWS_REGION}:*:launch-template/*",
+        "arn:${AWS_PARTITION}:ec2:${AWS_REGION}:*:spot-instances-request/*"
```

Comment:
This was technically added later (the v0.33 version) and isn't strictly necessary to run v0.32.x. I can go either way here. Seems fine to me to have expanded permissions here but also it isn't needed just to run the v0.32.x migration. Did this cause a problem at the v0.32.x transition for you or after upgrading to subsequent versions?

Reply:
Yes, it caused a problem at 0.32.x. After adding NodePools and EC2NodeClasss, karpenter started trying to provision using those but didn't have the correct permissions.

Reply:
Hmmm, that's odd since I was just able to run v0.32.10 without this permission and everything worked fine, including launching new NodeClaims with NodePools and EC2NodeClasses. Can you share the error that you are hitting that's causing you to think that you need this additional permission

Reply:
Yes, spot instances work fine as well. Can you share the error that you are hitting or a log dump so I can see what's going on and then maybe change some things to try and repro?